src/xdocs/cmdline.xml.vm

```diff
@@ -37,7 +37,8 @@ java -D&lt;property&gt;=&lt;value&gt;  \
      [-t | --tree] [-T | --treeWithComments] [-J | --treeWithJavadoc] [-j | --javadocTree] \
      [-V | --version] [-b | --branch-matching-xpath &lt;xpathQuery&gt;] [-h | --help] \
      [-e | --exclude &lt;excludedPath&gt;] [-E | --executeIgnoredModules] [-d | --debug] \
-     [-x | --exclude-regexp &lt;excludedPathPattern&gt;] \ file...
+     [-x | --exclude-regexp &lt;excludedPathPattern&gt;] \
+     file(s) or folder(s) ...
```

Comment:
This was probably copied from the actual command line which still says files.

````
> java -jar .\checkstyle-10.6.0-all.jar
Missing required parameter: '<files>'
Usage: checkstyle [OPTIONS]... FILES...
Try 'checkstyle --help' for more information.

> java -jar .\checkstyle-10.6.0-all.jar --help
Usage: checkstyle [-dEghjJtTV] [-b=<xpath>] [-c=<configurationFile>] [-f=<format>]
                  [-o=<outputPath>] [-p=<propertiesFile>] [-s=<suppressionLineColumnNumber>]
                  [-w=<tabWidth>] [-e=<exclude>]... [-x=<excludeRegex>]... <files>...
Checkstyle verifies that the specified source code files adhere to the specified rules. By default,
violations are reported to standard out in plain format. Checkstyle requires a configuration XML
file that configures the checks to apply.
      <files>...            One or more source files to verify
````